### PR TITLE
notificiation view implemented

### PR DESCRIPTION
**Pull request by iamavishkar**
## JIRA Ticket(s) if any

* [MOB-XXXX](https://iterable.atlassian.net/browse/MOB-XXXX)

## Test Steps

### CODE REVIEW

**Comment by Sagarpatel14** [comment]
image
image

File: src/components/notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    border: "none",
+    color: "#2196f3",
+    cursor: "pointer",
+  };
+
+  return (
+    <div style={cardStyle}>
+      <h2>{title}</h2>
+      <p>{description}</p>
+      {primaryButtonLabel ? (
```
Comment (mprew97) [inline src/components/notification/index.tsx:53]: nit: use `&&` here and remove the need for `: null`

File: src/components/notification/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        <button
+          onClick={onClickPrimaryBtn}
+          style={{ ...primaryButtonDefaultStyle, ...primaryButtonStyle }}
+        >
+          {primaryButtonLabel}
+        </button>
+      ) : null}
+      {secondaryButtonLabel ? (
```
Comment (mprew97) [inline src/components/notification/index.tsx:61]: nit: same as above

**Comment by iamavishkar** [comment]
request you to please review the PR as it has been updated.

**Comment by mprew97** [comment]
> request you to please review the PR as it has been updated.

Ty sir!

We should be able to just import the `Notification` component in `react-example` as a user would. Can we do that and also have an example of it being displayed on a page somewhere in the example? Like an actual live component example?

**Comment by devcsomnicg** [comment]
Hi @mprew97 here is the video for the same: https://drive.google.com/file/d/1ifip1OplImM9m2q-kOXGwxNwenqJXkCC/view?usp=sharing

**Pull request merged**